test.sh

```diff
@@ -187,6 +192,25 @@ then
     # Run Test UE
     cd test
     ${EXEC_UENS} $GOROOT/bin/go test -v -vet=off -timeout 0 -run TestNon3GPPUE -args noinit
+elif [[ "$1" == "TestMultiAmfRegistration" ]]
+then
+    # make -j
+    
+    # NF_LIST="nrf smf udr pcf udm nssf ausf"
+
+    # for NF in ${NF_LIST}; do
+    #     ./bin/${NF} -c ./config/${NF}cfg.yaml &
+    #     sleep 0.1
+    # done
+
+    ./bin/amf -c ./config/multiAMF/amfcfg.yaml &
```

Comment:
why not use new amf app to launch amf1/amf2 in code?

Reply:
It's because when calling `amfConfig()`, it will overwrite the `amf_factory.AmfConfig` which is shared by AMF1, AMF2. 

I think @ianchen0119 's method is not feasible because AMF should not be launched until NRF is launched.